src/Form/WebPageArchiveEditForm.php

```diff
@@ -18,58 +17,60 @@
 class WebPageArchiveEditForm extends WebPageArchiveFormBase {
 
 
-    /**
-     * The image effect manager service.
-     *
-     * @var \Drupal\web_page_archive\Plugin\CaptureUtilityManager
-     */
-    protected $captureUtilityManager;
-
-    /**
-     * Constructs an WebPageArchiveEditForm object.
-     *
-     * @param \Drupal\Core\Entity\EntityStorageInterface $web_page_archive_storage
-     *   The storage.
-     * @param \Drupal\web_page_archive\Plugin\CaptureUtilityManager $capture_utility_manager
-     *   The capture utility manager service.
-     */
-    public function __construct(EntityStorageInterface $web_page_archive_storage, CaptureUtilityManager $capture_utility_manager) {
-      parent::__construct($web_page_archive_storage);
-      $this->captureUtilityManager = $capture_utility_manager;
-    }
+  /**
+   * The capture utility manager service.
+   *
+   * @var \Drupal\web_page_archive\Plugin\CaptureUtilityManager
+   */
+  protected $captureUtilityManager;
 
-    /**
-     * {@inheritdoc}
-     */
-    public static function create(ContainerInterface $container) {
-      return new static(
-        $container->get('entity.manager')->getStorage('web_page_archive'),
-        $container->get('plugin.manager.capture_utility')
-      );
-    }
+  /**
+   * Constructs an WebPageArchiveEditForm object.
+   *
+   * @param \Drupal\Core\Entity\EntityStorageInterface $web_page_archive_storage
+   *   The storage.
+   * @param \Drupal\web_page_archive\Plugin\CaptureUtilityManager $capture_utility_manager
+   *   The capture utility manager service.
+   */
+  public function __construct(EntityStorageInterface $web_page_archive_storage, CaptureUtilityManager $capture_utility_manager) {
+    parent::__construct($web_page_archive_storage);
+    $this->captureUtilityManager = $capture_utility_manager;
+  }
 
   /**
    * {@inheritdoc}
    */
-  public function form(array $form, FormStateInterface $form_state) {
-    $form = parent::form($form, $form_state);
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity.manager')->getStorage('web_page_archive'),
```

Comment:
`entity.manager` is deprecated.  We should use `entity_type.manager` instead (standard operations) or `entity.repository` instead (loading by uuid).  I haven't looked to see which context makes sense yet.